addon/components/tf-input-group/template.hbs

```diff
@@ -0,0 +1,18 @@
+{{yield (hash
+  label=(component 'tf-input/input-label' 
+          sizeStyle=sizeStyle 
```

Comment:
Anyone have an opinion on indentation for stuff like this?? (Such nitpicking! Sorry!)